app/src/main/java/com/napzak/market/presentation/explore/type/SortType.kt

```diff
@@ -0,0 +1,12 @@
+package com.napzak.market.presentation.explore.type
+
+enum class SortType(val label: String) {
+    RECENT(label = "최신순"),
+    POPULAR(label = "인기순"),
+    HIGH_PRICE(label = "고가순"),
+    LOW_PRICE(label = "저가순"),
+}
+
+fun getSortLabel(type: String): String {
+    return SortType.entries.find { it.name == type }?.label ?: "잘못된 SortType 값 입니다."
+}
```

Comment:
companion object에 넣어서 SortType 내부로 함수를 옮깁시다!
```suggestion
enum class SortType(val label: String) {
    RECENT(label = "최신순"),
    POPULAR(label = "인기순"),
    HIGH_PRICE(label = "고가순"),
    LOW_PRICE(label = "저가순"),

    companion object {
        fun getSortLabel(type: String): String {
            return entries.find { it.name == type }?.label ?: "잘못된 SortType 값 입니다."
        }
    }
}
```